src/neo/Network/P2P/Payloads/GetBlockDataPayload.cs

```diff
@@ -4,16 +4,17 @@
 
 namespace Neo.Network.P2P.Payloads
 {
-    public class GetBlockByIndexPayload : ISerializable
+    public class GetBlockDataPayload : ISerializable
```

Comment:
Why modify this?

Reply:
Should `GetBlockByIndexPayload` as well as corresponding transmission logic be kept?

Reply:
The logic is not change. I only saw you changed its name.

Reply:
`count` is changed from `short` back to `ushort`. We can keep the name `GetBlockByIndexPayload` but still logic will be different.

Reply:
But why change it from `short` to `ushort`?

Reply:
Beforehand this value is allowed to be -1 for this scenario:
https://github.com/neo-project/neo/blob/677bd9648d40b31076ba78c88c8a48bc0e919f66/src/neo/Network/P2P/RemoteNode.ProtocolHandler.cs#L197
Should we keep this logic?

Reply:
Yes.

Reply:
Done